appserver/admingui/web/src/main/resources/org/glassfish/web/admingui/Strings.properties

```diff
@@ -300,6 +300,8 @@ http.XpoweredBy=XPowered By:
 http.XpoweredByHelp=Include X-Powered-By: Servlet/3.0 in servlet-generated HTTP response headers
 http.ServerHeader=Server Header:
 http.ServerHeaderHelp=Include Server Header: Servlet/3.0 in servlet-generated HTTP response headers
+http.XframeOptions=XFrame Options:
+http.XframeOptionsHelp=Include X-Frame-Options: Servlet/3.0 in servlet-generated HTTP response headers
```

Comment:
Probably should say "Default X-Frame-Options value to be included in all responses"